tasks.py.jinja

```diff
@@ -283,13 +286,27 @@ def lint(c, verbose=False):
 
 
 @task(develop)
-def start(c, detach=True, debugpy=False):
+def start(c, detach=True, debugpy=False, wait=None):
     """Start environment."""
     cmd = "docker-compose up"
     if detach:
         cmd += " --detach"
     with c.cd(str(PROJECT_ROOT)):
         c.run(cmd, env=dict(UID_ENV, DOODBA_DEBUGPY_ENABLE=str(int(debugpy))))
+    if wait:
```

Comment:
if it helps, a snippet of a function I use to wait for Pycharm debug to be open. Could be adjusted to work in a while loop.

```python
    import socket

    def socket_is_open(host, port):
        sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
        if sock.connect_ex((host, port)) == 0:
            return True
        return False
```